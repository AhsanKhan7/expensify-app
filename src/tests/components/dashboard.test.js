import React from 'react'
import { shallow } from 'enzyme' 
import dashboardPage from '../../components/Dashboard'

test('should dashboard component run', () => {
    const wrapper = shallow(<dashboardPage />)
    expect(wrapper).toMatchSnapshot() 
})