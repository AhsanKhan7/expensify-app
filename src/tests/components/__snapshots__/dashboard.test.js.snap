// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should dashboard component run 1`] = `<dashboardPage />`;
